tests/schema_/test_embedded_model.py

```diff
@@ -627,3 +630,38 @@ class Meta:
             )
             editor.delete_model(Author)
         self.assertTableNotExists(Author)
+
+
+class EmbeddedModelsIgnoredTests(TestMixin, TransactionTestCase):
+    def test_embedded_not_created(self):
+        """create_model() and delete_model() ignore EmbeddedModel."""
+        with connection.schema_editor() as editor:
+            editor.create_model(Book)
+            editor.create_model(Address)
+            editor.create_model(Author)
+            self.assertTableExists(Book)
+            self.assertTableNotExists(Address)
+            self.assertTableNotExists(Author)
+            editor.delete_model(Book)
+            with self.assertNumQueries(0):
+                editor.delete_model(Address)
+                editor.delete_model(Author)
+        self.assertTableNotExists(Book)
+
+    def test_embedded_add_field_ignored(self):
+        """add_field() and remove_field() ignore EmbeddedModel."""
+        new_field = models.CharField(max_length=1, default="a")
+        new_field.set_attributes_from_name("char")
+        with connection.schema_editor() as editor, self.assertNumQueries(0):
+            editor.add_field(Author, new_field)
+        with connection.schema_editor() as editor, self.assertNumQueries(0):
+            editor.remove_field(Author, new_field)
+
+    def test_embedded_alter_field_ignored(self):
+        """alter_field() ignores EmbeddedModel."""
+        old_field = models.CharField(max_length=1)
+        old_field.set_attributes_from_name("old")
+        new_field = models.CharField(max_length=1)
+        new_field.set_attributes_from_name("new")
+        with connection.schema_editor() as editor, self.assertNumQueries(0):
+            editor.alter_field(Author, old_field, new_field)
```

Comment:
A little confused here. Why would we want them ignored?

Reply:
While we still don't detect changes to embedded models as part of the parent model, `makemigrations` still generates a migration as if the EmbeddedModel has its own collection and we need to ignore that rather than trying to run some change on a nonexistent collection.